.circleci/config.yml

```diff
@@ -269,31 +229,66 @@ jobs:
               -destination 'platform=iOS Simulator,name=iPhone 11' \
               -scheme ApplePlatformsIntegrationMobileTests
           working_directory: test/ApplePlatformsIntegrationTestApp
+
+  build-apple-runtime:
+    <<: *apple_defaults
+    steps:
+      - checkout
+      - restore_cache:
+          key: v3-repo-{{ .Environment.CIRCLE_PR_NUMBER }}
+      - run:
+          name: Set up workspace
+          command: mkdir -p /tmp/hermes/output
+      - run:
+          name: Install dependencies
+          command: |
+            brew install cmake
+            sudo gem install cocoapods
+      - run:
+          name: Build the iOS frameworks
+          command: ./utils/build-ios-framework.sh
+      - run:
+          name: Build the Mac frameworks
+          command: ./utils/build-mac-framework.sh
+      - run:
+          name: Build the CLI for Mac
+          command: cmake --build ./build_macosx --target github-cli-release
       - save_cache:
-          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          key: v3-repo-{{ .Environment.CIRCLE_PR_NUMBER }}
           paths:
-            - ~/hermes
+            - ~/hermes/build_iphoneos
+            - ~/hermes/build_iphonesimulator
+            - ~/hermes/build_macosx
+            - ~/hermes/destroot
 
-  apple-runtime:
+  package-apple-runtime:
     <<: *apple_defaults
     steps:
+      - checkout
       - restore_cache:
-          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          key: v3-repo-{{ .Environment.CIRCLE_PR_NUMBER }}
       - run:
           name: Set up workspace
           command: mkdir -p /tmp/hermes/output
       - run:
           name: Install dependencies
           command: |
             brew install cmake ninja
+            sudo gem install cocoapods
       - run:
           name: Package the framework
           command: |
-            cmake --build ./build_iphoneos --target hermes-runtime-darwin-cocoapods-release
+            . ./utils/build-apple-framework.sh
+
+            mkdir -p /tmp/cocoapods-package-root/destroot
+            cp -R ./destroot /tmp/cocoapods-package-root/destroot
+            cp hermes-engine.podspec LICENSE /tmp/cocoapods-package-root
+
+            tar -C /tmp/cocoapods-package-root/ -czvf /tmp/hermes/output/hermes-runtime-darwin-v$(get_release_version).tar.gz .
       - run:
-          name: Copy artifacts
+          name: Package the CLI
           command: |
-            cp "build_iphoneos/github"/hermes-runtime-darwin-*.tar.gz /tmp/hermes/output
```

Comment:
This was a bug from previous PR: wrong build folder was supplied for the CLI tools.

Reply:
CC: @alloy 

I believe your PR fixed it for local builds, still leaving this error on the CI: https://github.com/facebook/hermes/pull/354